### ISSUE TITLE
Join "Differences Between HTTP Entities and RFC 2045 Entities"? (Trac #118)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#118]
"Differences Between HTTP Entities and RFC 2045 Entities" currently is split between Part 1, Appendix C, and Part 3, Appendix A.

Part 1, Appendix C, talks about date formats, all the remaining differences currently are described in Part 3.

As Part 1 currently doesn't refer (internally) to Appendix C at all, but *does* refer to Part 3, Appendix A, we may want to consider to move the one subparagraph back into Part 3.

Migrated from https://trac.ietf.org/ticket/118
```json
{
    "status": "closed",
    "changetime": "2012-03-22T00:05:40",
    "_ts": "1332374740142649",
    "description": "\"Differences Between HTTP Entities and RFC 2045 Entities\" currently is split between Part 1, Appendix C, and Part 3, Appendix A.\n\nPart 1, Appendix C, talks about date formats, all the remaining differences currently are described in Part 3.\n\nAs Part 1 currently doesn't refer (internally) to Appendix C at all, but *does* refer to Part 3, Appendix A, we may want to consider to move the one subparagraph back into Part 3.\n",
    "reporter": "julian.reschke@gmx.de",
    "cc": "",
    "resolution": "fixed",
    "time": "2008-06-04T14:35:41",
    "component": "p1-messaging",
    "summary": "Join \"Differences Between HTTP Entities and RFC 2045 Entities\"?",
    "priority": "normal",
    "keywords": "",
    "milestone": "06",
    "owner": "",
    "type": "editorial",
    "severity": "Active WG Document"
}
```

**Comment by mnot** [comment]
Trac update at 20081114T01:22:01: 
* mnot@pobox.com commented:

> Yes.


* mnot@pobox.com changed milestone from "unassigned" to "06"

**Comment by mnot** [comment]
Trac update at 20081114T20:26:23: 
* julian.reschke@gmx.de commented:

> Fixed in [385]:
> 
> Resolve #118: move "Conversion of Date Formats" to old position inside "Differences Between HTTP Entities and RFC 2045 Entities" (Part 3) (closes #118)


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"

**Comment by mnot** [comment]
Trac update at 20120322T00:05:40: 
* mnot@pobox.com changed priority from "" to "normal"
* mnot@pobox.com changed severity from "Candidate WG Document" to "Active WG Document"
